test/tests/functional/pbs_test_tpp.py

```diff
@@ -374,20 +377,25 @@ def test_comm_with_vnode_insertion(self):
         for vnode in vnode_list:
             self.server.expect(VNODE, a, id=vnode)
 
-    def common_setup(self):
+    def common_setup(self, no_mom_on_comm=False, no_comm_on_server=False):
         """
         This function sets the shortnames of moms and comms in the cluster
         accordingly.
         Mom objects : self.momA, self.momB, self.momC
         Mom shortnames : self.hostA, self.hostB, self.hostC
```

Comment:
not sure if these docstrings are helping. A brief about how the setup is done will help.